skins/laika/src/components/DonationCommentPopUp.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+	<form id="laika-comment" v-on:submit.prevent="postComment" method="post" ref="form" class="modal-card">
+		<input type="hidden" name="donationId" :value="confirmationData.donation.id"/>
+		<input type="hidden" name="updateToken" :value="confirmationData.donation.updateToken">
+		<p class="modal-card-title has-margin-bottom-18">{{ $t( 'donation_comment_popup_title' ) }}</p><br>
+		<p class="has-margin-bottom-18">{{ $t( 'donation_comment_popup_explanation' ) }}</p>
+		<b-field :label="$t( 'donation_comment_popup_label' )">
+			<b-input id="comment" name="comment" type="textarea"></b-input>
```

Comment:
This `b-field` is stopping the `textarea` element from being the full width of the pop-up (like in Figma). The only thing I can think of would be to get rid of the `b-field` and instead just create a div with bottom margin and label yourself.